scheds/rust/scx_chaos/src/bpf/main.bpf.c

```diff
@@ -78,6 +82,9 @@ static __always_inline enum chaos_trait_kind choose_chaos(struct chaos_task_ctx
 	if (bpf_get_prandom_u32() < random_delays_freq_frac32)
 		return CHAOS_TRAIT_RANDOM_DELAYS;
 
+	if (bpf_get_prandom_u32() < cpu_freq_frac32)
+		return CHAOS_TRAIT_CPU_FREQ;
+
```

Comment:
my plan here was to take one random number, then if each didn't match do `-= random_delays_freq_frac32` (or whatever) and compare the next. If that can't look neat though let's stick to multiple randoms.